src/js/components/SurveyCollection.js

```diff
@@ -350,11 +350,22 @@ class SurveyQuestionTree extends React.Component {
                     >
                         {this.state.showAnswers ? <span>-</span> : <span>+</span>}
                     </button>
+
+                    {this.props.getRulesById(this.props.surveyNode.id).length > 0
```

Comment:
You dont need a ternary here.  Use `this.props.getRulesById(this.props.surveyNode.id).length > 0 &&`

Reply:
Done made the change and added icon!